src/crimson/osd/osd_operations/peering_event.cc

```diff
@@ -92,8 +90,8 @@ seastar::future<> PeeringEvent::start()
     }
   }).then([this] {
     return shard_services.send_pg_temp();
-  }).then([this, ref=std::move(ref)] {
-    logger().debug("{}: complete", *this);
+  }).finally([ref=std::move(ref)] {
+    logger().debug("{}: complete", *ref);
```

Comment:
we meant to print the ref?

Reply:
@ronen-fr `ref` is an instance of `intrusive_ptr<PeeringEvent>`, before this change we print `*this`, after this change, we print `*ref`, according to https://www.boost.org/doc/libs/1_75_0/libs/smart_ptr/doc/html/smart_ptr.html#intrusive_ptr

> `T & operator*() const noexcept;

`*ref` returns a reference of `PeeringEvent`. so the behavior is not changed.